packages/moonstone/RangePicker/RangePicker.js

```diff
@@ -120,38 +118,37 @@ const RangePickerBase = kind({
 		incrementIcon: PropTypes.string,
 
 		/**
-		 * Determines the user interaction of the control. A joined picker allows the user to use
-		 * the arrow keys to adjust the picker's value. The user may no longer use those arrow keys
-		 * to navigate, while this control is focused. A split control allows full navigation,
-		 * but requires individual ENTER presses on the incrementer and decrementer buttons.
-		 * Pointer interaction is the same for both formats.
+		 * Indicates that users can use the arrow keys to adjust the picker's value.
+		 * The user may no longer use those arrow keys to navigate, while this control is focused.
+		 * A default control allows full navigation, but requires individual ENTER presses on the incrementer
+		 * and decrementer buttons. Pointer interaction is the same for both formats.
 		 *
 		 * @type {Boolean}
 		 * @public
 		 */
 		joined: PropTypes.bool,
 
 		/**
-		 * By default, the picker will animate transitions between items if it has a defined
-		 * `width`. Specifying `noAnimation` will prevent any transition animation for the
-		 * component.
+		 * By default, the picker will animate transitions between items.
```

Comment:
Let's reword this so the function of the property is above.  An additional sentence after the blank line could expand upon it.